roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -323,6 +323,13 @@ openstack_lbaas_monitor_timeout: "30s"
 openstack_lbaas_monitor_max_retries: "3"
 openstack_cacert: "{{ lookup('env','OS_CACERT') }}"
 
+# Default values for the external OpenStack Cloud Controller
+external_openstack_lbaas_use_octavia: true
```

Comment:
I think the default from OpenStack is to not use Octavia. https://github.com/kubernetes/cloud-provider-openstack/blob/2d764b22db7711d19d35d3e40adac4ca7023f014/docs/provider-configuration.md#load-balancer

Might want to keep that here as well.

Reply:
I have put Octavia as default because Neutron LBaaS is deprecated since Queens: https://docs.openstack.org/api-ref/network/v2/index.html#load-balancer-as-a-service-2-0-deprecated

Reply:
Ah, missed that. Makes sense to have it as default then.